.travis.yml

```diff
@@ -32,3 +32,13 @@ before_script:
 script: ./gradlew lint connectedCheck
 
 sudo: false
+
+before_deploy: ./gradlew javadoc
+
+deploy:
+  provider: pages
+  local_dir: docs # only include the contents of the docs dir
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN # Set in travis-ci.org dashboard
+  on:
+    branch: master # deploy when changes pushed to this branch
```

Comment:
This should be on tags instead of master to ensure only released content is being documented